sub_cal/sub_cal/src/main/java/model/Operation.java

```diff
@@ -0,0 +1,72 @@
+package model;
+
+import java.math.BigDecimal;
+import java.util.Stack;
+import java.util.StringTokenizer;
+public class Operation {
+    private Stack<Double> numberStack = new Stack<>();
+    private Stack<String> operStack = new Stack<>();
+    private static final int CONVERTTONEGATIVE = -1;
```

Comment:
```suggestion
    private static final int CONVERT_TO_NEGATIVE = -1;
```

자바 상수 네이밍에서는 띄어쓰기의 경우 `_`로 구분합니다.

Reply:
앗 알겠습니다 
해당 부분도 조심하겠습니다